safe/gui/tools/help/peta_bencana_help.py

```diff
@@ -81,9 +82,9 @@ def content():
         'This tool requires a working internet connection and fetching '
         'data will consume your bandwidth.'))
     tips.add(m.Link(
-        'https://petajakarta.org/banjir/en/data/',
+        production_api['help_url'],
         text=tr(
-            'Downloaded data is copyright the PetaJakarta contributors'
+            'Downloaded data is copyright the PetaBencana contributors'
```

Comment:
And here `PetaBencana`. According to their website https://docs.petabencana.id they always use one word `Petabencana` except one time `PetaBencana.id`. Maybe we should use `Petabencana`, what do you think ?Can you please uniform words ?